### PR TITLE
Test the role via molecule and update supported OS

### LINKED ISSUE #8: Test role via Molecule

### PR DESCRIPTION
**Pull request by tobiashuste**
* Manage dependencies via pipenv
* Run Molecule via GitHub Actions
* Update the list of support OS

Closes #8

### CODE REVIEW

**Comment by tobiashuste** [comment]
@Normo I would be happy if you could have a look. This implements basic role testing via molecule. There is no verification step implemented yet. It simply takes the default installation and deploys it in one of the container.

**Review by Normo — CHANGES_REQUESTED** [review]
Thank you very much @tobiashuste for implementing this! :star_struck: Very much appreciate the effort. I've added some comments/suggestions.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+name: "CI"
+on:
+  pull_request:
+  push:
+    branches:
+      - "main"
```
[the remark this answers is not in the document]
Comment (Normo) [reply .github/workflows/ci.yml:7]: What do you think @tobiashuste ? Shall we rename the default branch to `main`?

**Comment by Normo** [comment]
@tobiashuste We can take inspiration for the verfication step from the test play in [tests/test.yml](https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/master/tests/test.yml#L28)

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tobiashuste) [reply .github/workflows/ci.yml:7]: Probably we should do this.

**Comment by tobiashuste** [comment]
> @tobiashuste We can take inspiration for the verfication step from the test play in [tests/test.yml](https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/master/tests/test.yml#L28)

@Normo I implemented the verification step according to the existing tests. I also changed your other suggestions.

So far I did not touch the branch name to wait for your feedback.

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Normo) [reply .github/workflows/ci.yml:7]: :ok_hand: let's do it with #13

**Review by Normo — APPROVED** [review]
Thanks, @tobiashuste LGTM :+1:

**Pull request merged**